lib/SauceLabs.js

```diff
@@ -268,6 +270,9 @@ function generateLink(hostname, auth, date, job) {
 }
 
 function makeRequest(options, body, callback) {
+    if(process.env.https_proxy !=='undefined' && process.env.https_proxy) {
```

Comment:
Please could you convert `process.env.https_proxy` to `process.env.HTTPS_PROXY` as this is the standard convention when defining environment variables.